src/Tribe/PUE/Package_Handler.php

```diff
@@ -70,9 +70,9 @@ public function filter_upgrader_pre_download( $reply, $package, WP_Upgrader $upg
 	 * @return bool
 	 */
 	protected function is_mt_package( $package ) {
-		if ( empty( $package )
-		     || ! preg_match( '!^(http|https|ftp)://!i', $package )
-		     || file_exists( $package )
```

Comment:
The removal of this line gives me pause. Your PR description reads:

> If `$package` is empty, or is not a remote URL, returns false. Seems there's need to do a further confirmation check by testing if the file exists (which tries to use stream wrappers when available).

Shouldn't we continue to handle the `file_exists()` case then?